pkgs/development/tools/minipro/default.nix

```
@@ -0,0 +1,44 @@
+{stdenv, fetchgit, pkgs, pkg-config}:
+stdenv.mkDerivation rec {
+  name = "minipro-0.4";
```

Should be using `pname` and `version`.
```suggestion
  pname = "minipro";
  version = "0.4";
```